.clang-format

```diff
@@ -51,7 +51,7 @@ SpaceAfterCStyleCast: false
 BreakBeforeBraces: Custom
 
 # Control of individual brace wrapping cases
-BraceWrapping: {
+BraceWrapping:
```

Comment:
This change makes this valid yaml so you can use it locally with `ament_clang_format`.